### PR TITLE
Added python-env and documentation about open-ce-python

### PR DESCRIPTION
**Pull request by npanpaliya**
## Checklist before submitting

- [ ] Did you read the [contributor guide](https://github.com/open-ce/open-ce/blob/main/CONTRIBUTING.md)?
- [ ] Did you update any affected [documentation](https://github.com/open-ce/open-ce/blob/main/doc/)?
- [ ] Did you write any tests to validate this change?

## Description

Added python-env.yaml to build optimized python. I'm still working on adding github actions for this.

## Review process to land 

1. All tests and other checks must succeed.
2. At least one [maintainer](https://github.com/open-ce/open-ce/blob/main/MAINTAINERS.md) must review and approve.
3. If any  [maintainer](https://github.com/open-ce/open-ce/blob/main/MAINTAINERS.md) requests changes, they must be addressed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,47 @@
...
+
+## GETTING STARTED
+
+### Requirements
+
+* `conda`
+  * The conda tool can either be installed through [Anaconda](https://www.anaconda.com/products/individual#Downloads) or [Miniconda](https://docs.conda.io/en/latest/miniconda.html).
+
+* `Redhat GCC 11`
+  Redhat's `gcc-toolset-11` package is needed which can be installed via `yum install -y gcc-toolset-11`. To install this, we need to have RHEL 8.x with Power9/Power10 architecture.
```
Comment (alexlang74) [inline README.md]: ~To install this, we need to have RHEL 8.x with Power9/Power10 architecture.~  REHL 8.x with Power 9 or Power 10 architecture is required.

**Review by alexlang74 — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (npanpaliya) [reply README.md]: Done. Thanks.

File: README.md [hunk 2]
```diff
@@ -1 +1,47 @@
...
+Open-CE Python can be built like any other Open-CE package. This repository contains `envs/python-env.yaml` which is to be used to build optimized python. The `open-ce` tool can be used to build this python. For more information on the `open-ce` tool, please see the open-ce-builder [repository](https://github.com/open-ce/open-ce-builder).
+
+For example:
+```shell
+    open-ce build env --python_versions=3.10 --ppc_arch=p10 python-env.yaml
+```
+
+## Contributions
+
+For contribution information, please see the [CONTRIBUTING.md](CONTRIBUTING.md) page.
```
Comment (cdeepali) [inline README.md]: I think we can link to the OpenCE Contributing page here.

File: README.md [hunk 3]
```diff
@@ -1 +1,47 @@
...
+</p>
+
+[![Python Support](https://img.shields.io/badge/python-3.10-blue.svg)](#requirements)
+[![Architecture Support](https://img.shields.io/badge/architecture-ppc64le-blue)](#)
+[![GitHub Licence](https://img.shields.io/github/license/open-ce/open-ce.svg)](LICENSE)
+---
+
+This is a sub-project of the Open-CE project for building python optimized for Power10 architecture.
+
+This repository contains an Open-CE environment file that builds optimized python using the [python feedstock](https:///github.com/open-ce/python-feedstock). Open-CE-Python currently supports the following:
```
Comment (cdeepali) [inline README.md]:
```suggestion
This repository contains an Open-CE environment file that builds optimized python using the [python feedstock](https:///github.com/open-ce/python-feedstock). Open-CE Python currently supports the following:
```

File: README.md [hunk 4]
```diff
@@ -1 +1,47 @@
...
+
+* `conda`
+  * The conda tool can either be installed through [Anaconda](https://www.anaconda.com/products/individual#Downloads) or [Miniconda](https://docs.conda.io/en/latest/miniconda.html).
+
+* `Redhat GCC 11`
+  Redhat's `gcc-toolset-11` package is needed which can be installed via `yum install -y gcc-toolset-11`. RHEL 8.x with Power 9 or Power 10 architecture is required.
+
+### Building Open-CE Python
+
+Open-CE Python can be built like any other Open-CE package. This repository contains `envs/python-env.yaml` which is to be used to build optimized python. The `open-ce` tool can be used to build this python. For more information on the `open-ce` tool, please see the open-ce-builder [repository](https://github.com/open-ce/open-ce-builder).
```
Comment (cdeepali) [inline README.md]:
```suggestion
Open-CE Python can be built like any other Open-CE package. This repository contains `envs/python-env.yaml` which is to be used to build optimized python. The `open-ce` tool can be used to build this python. For more information on the `open-ce` tool, please see the Open-CE Builder [repository](https://github.com/open-ce/open-ce-builder).
```

**Review by cdeepali — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (npanpaliya) [reply README.md]: Done

**Pull request merged**